tests/codegen/simd/unpadded-simd.rs

```diff
@@ -14,6 +14,6 @@ pub struct int16x4x2_t(pub int16x4_t, pub int16x4_t);
 
 // CHECK: %int16x4x2_t = type { <4 x i16>, <4 x i16> }
 #[no_mangle]
-fn takes_int16x4x2_t(t: int16x4x2_t) -> int16x4x2_t {
+extern "unadjusted" fn takes_int16x4x2_t(t: int16x4x2_t) -> int16x4x2_t {
     t
 }
```

Comment:
This test only incidentally used the struct type (via `byval`, so it would be removed by these changes), but the original motivation (#87254) was for the `unadjusted` ABI, where we use the struct type directly and pass the vectors by value. Changed it to test that.